pkgs/tools/security/vault/default.nix

```diff
@@ -7,27 +7,32 @@ let
     rev = "e2f59b64be1fa5430fa05c91b6274284de4ea77c";
     sha256 = "10m75rp3hy71wlmnd88grmpjhqy0pwb9m8wm19l0f463xla54frd";
   };
-in buildGoPackage rec {
+in stdenv.mkDerivation rec {
```

Comment:
Unlike a typical Go project, Vault sources have a Makefile.

Vault works being compiled with ```buildGoPackage```, but some functions fail, for example:
```
$ vault version   
Vault (version unknown) (cgo)
```